tasks/taxon_id/contamination/task_kraken2.wdl

```diff
@@ -129,6 +57,21 @@ task kraken2_standalone {
     if [ -z "$percentage_human" ] ; then percentage_human="0" ; fi
     echo $percentage_human | tee PERCENT_HUMAN
     
+    # Report percentage of SARS-CoV-2 reads
+    percentage_sc2=$(grep "Severe acute respiratory syndrome coronavirus 2" ~{samplename}.report.txt | cut -f1 )
```

Comment:
I'm proposing a change of behaviour. The proportion of SC2 is always reported regardless of the organisms. This doesn't make much sense in an amplicon-based workflow. I propose we remove this block and report the proportion of SC2 if the target organism is indeed SC2. This block of code would be removed and the target_organism string would be added to the wf_organism_parameters subworkflow. @sage-wright and @AndrewLangvt what do you think?

Reply:
I'm in favor of only reporting it for SC2, but I think we should keep the output name the same. Just wrap that section into an `if org == "sc2"` block and make the output optional

Reply:
I'll be following your suggestion! Thank you!